src/commcare_cloud/commands/terraform/constants.py

```diff
@@ -55,5 +56,5 @@
 """.strip().split('\n')
 
 COMMCAREHQ_XML_QUERYSTRING_URLS_REGEX = """
```

Comment:
I would suggest fixing the lint error not by escaping the `\` within the string, but by making this an r-string (starting with `r"""`) as described here https://docs.python.org/3/library/re.html:
> The solution is to use Python’s raw string notation for regular expression patterns; backslashes are not handled in any special way in a string literal prefixed with 'r'. So r"\n" is a two-character string containing '\' and 'n', while "\n" is a one-character string containing a newline. Usually patterns will be expressed in Python code using this raw string notation.

That's also how the block of regular expressions above is defined.